src/app/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js

```diff
@@ -142,15 +262,19 @@ export class MyAccountMyWishlist extends PureComponent {
         const {
             isWishlistLoading,
             isWishlistEmpty,
-            isLoading
+            isLoading,
+            isMobile
         } = this.props;
+        const { actionLineHeight } = this.state;
 
         if (isWishlistEmpty && !isWishlistLoading) {
             return this.renderNoProductsFound();
         }
 
+        const blockStyles = isMobile ? { marginBottom: actionLineHeight } : {};
+
         return (
-            <div block="MyAccountMyWishlist" elem="Products">
+            <div block="MyAccountMyWishlist" elem="Products" style={ blockStyles }>
```

Comment:
Why this actionLineHeight + blockStyles magic cannot be achieved with SCSS? Why keep this logic in JS?